Boards/mfboard.schema.json

```diff
@@ -264,6 +264,39 @@
               "type": "string"
             }
           }
+        },
+        "DeviceConfigs": {
+          "$id": "#root/Info/DeviceConfigs",
+          "title": "DeviceConfigs",
+          "description": "A list of default device configs for this board.",
+          "type": "array",
+          "default": [],
+          "items": {
```

Comment:
Name, Description, and File are all required, yes? They should be listed as required in the schema.